packages/react-core/src/components/Button/examples/Button.md

```diff
@@ -16,68 +16,99 @@ import { Link } from '@reach/router';
+| Plain | Plain buttons have no styling and are intended to be labeled with icons. |
+| Control | Control buttons can be labeled with text or icons. Primarily intended to be paired with other controls in an [input group](https://www.patternfly.org/v4/components/input-group). | 
+
+### Disabled buttons
+
+To indicate that an action is currently unavailable, all button variations can be disabled using the `isDisabled` property.
 
 ```ts file="./ButtonDisabled.tsx"
 ```
 
-### Aria disabled
+### Small buttons
 
-```ts file="./ButtonAriaDisabled.tsx"
+To fit into tight spaces, primary, secondary, tertiary, danger, and warning button variations can be made smaller using the `isSmall` property.
+
+```ts file="./ButtonSmall.tsx"
 ```
 
-### Aria disabled button with tooltip
+### Call to action (CTA) buttons
 
-```ts file="./ButtonAriaDisabledTooltip.tsx"
+CTA buttons and links direct users to complete an action. Primary, secondary, tertiary, and link button variants can be styled as CTAs using the `isLarge` property. 
+
+```ts file="./ButtonCallToAction.tsx"
 ```
 
-### Aria disabled link as button with tooltip
+### Block level buttons
 
-```ts file="./ButtonAriaDisabledLinkTooltip.tsx"
+Block level buttons span the full width of the parent element and can be enabled using the `isBlock` property.
+
+```ts file="./ButtonBlock.tsx"
+```
+
+### Progress indicators
+
+Progress indicators can be added to buttons to identify that an action is in progress after a click. 
+
+```ts file="./ButtonProgress.tsx"
 ```
 
 ### Links as buttons
 
+Buttons that link to another resource may take the form of primary, secondary, tertiary, or link variants. Use `component="a"` and an `href` property to designate the button's target link.
+
 ```ts file="./ButtonLinks.tsx"
 ```
 
 ### Inline link as span
 
+Inline links should use `component="span"` and the `isInline` property to wrap inline with surrounding text.
+
 ```ts file="./ButtonInlineSpanLink.tsx"
 ```
 
-### Block level
-
-```ts file="./ButtonBlock.tsx"
-```
+### Router link
 
-### Types
+Router links can be used for in-app linking in React environments to prevent page reloading.
 
-```ts file="./ButtonTypes.tsx"
+```ts file="./ButtonRouterLink.tsx"
 ```
 
-### Small
+### Aria-disabled examples
 
-```ts file="./ButtonSmall.tsx"
-```
+[Accessible Rich Internet Applications (ARIA)](https://www.w3.org/WAI/standards-guidelines/aria/) is a set of roles and attributes specified by the [World Wide Web Consortium]. ARIA defines ways to make web content and web applications more accessible to people with disabilities.
 
-### Call to action
+Buttons that are aria-disabled are similar to normal disabled buttons, except they can receive focus. Every button variant can be aria disabled using the `isAriaDisabled` property.
```

Comment:
I'm probably wrong but should these have the same hyphenation (either with or without)? Seems like if they "are" or "can be" a thing, it's the same thing, and it's "aria-disabled" everywhere else<sup>*</sup>.

```suggestion
Buttons that are aria-disabled are similar to normal disabled buttons, except they can receive focus. Every button variant can be aria-disabled using the `isAriaDisabled` property.
```

<sup>*</sup> except for the [aria-disabled button examples](https://patternfly-react-pr-7607.surge.sh/components/button#aria-disabled-examples) - most say "aria disabled", though some just say "disabled" (even though they're aria-disabled). That may be beyond the scope of this PR tho.

Reply:
For now, I opted to just match everything as aria-disabled in my latest push (at least within the aria section), but that can be changed if it makes sense to do something different!